user/xournalpp/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "xournalpp"
+pkgver = "1.2.3"
+pkgrel = 0
+archs = ["aarch64", "x86_64"]
```

Comment:
i don't see why this is arch limited

Reply:
actually, i saw s390x, riscv64 and loongaarch64 being blacklisted over at aports so i went with those two. i haven't tested the rest of the architectures myself though so I guess it's better to delete the constraints